tutorials/cloud-iot-coap-proxy/coap-dtls-client/client.env

```diff
@@ -0,0 +1,26 @@
+#! /bin/bash
```

Comment:
consider adding a copyright / license header to your shell scripts and other code that doesn't have a header

Reply:
FWIW - the community writing guidelines do not prescribe that (maybe it should?) - I'll do that for googler authored stuff in general, this was contractor code as we discussed in chat - I'm going to consider this is covered by CLA